source/local/local.go

```diff
@@ -280,6 +298,27 @@ func getFeaturesFromFiles() (map[string]string, error) {
 	return features, nil
 }
 
+// Return the expiration date of a feature file
+func getExpirationDate(lines [][]byte) (time.Time, error) {
+	for _, line := range lines {
+		if len(line) > 0 {
+			lineSplit := strings.SplitN(string(line), ":", 2)
+
+			key := lineSplit[0]
+			if key == ExpiryDateKey {
+				expiryDate, err := time.Parse(time.RFC3339, strings.TrimSpace(lineSplit[1]))
+				fmt.Println(err)
+				if err != nil {
+					return time.Now(), err
+				}
+				return expiryDate, nil
+			}
+		}
+	}
+
+	return time.Now(), nil
```

Comment:
I think this is bad as it basically makes all files that don't have the `+expiry-time` directive expired (as this timestamp is checked like 1 microsecond after `Now()` and it has already passed)